packages/mui-material/src/TablePagination/TablePaginationActions.d.ts

```diff
@@ -23,6 +28,12 @@ export interface TablePaginationActionsProps extends React.HTMLAttributes<HTMLDi
   rowsPerPage: number;
   showFirstButton: boolean;
   showLastButton: boolean;
+  slotProps?: {
```

Comment:
We should deprecate `backIconButtonProps` and `nextIconButtonProps`, same way we did on `TablePagination`